src/view/ChewingEditor.cpp

```diff
@@ -128,6 +130,57 @@ void ChewingEditor::exportUserphrase(const QString& file)
     emit model_->exportUserphrase(exporter);
 }
 
+void ChewingEditor::showAbout()
+{
+/* Make about dialog shows like aboutQt() on Mac OS X.
+   So we have to prevent call twice on Mac OS X. */
+#ifdef Q_OS_MAC
+    static QPointer<QMessageBox> oldAboutBox;
+
+    if (oldAboutBox) {
+        oldAboutBox->show();
+        oldAboutBox->raise();
+        oldAboutBox->activeWindow();
+        return;
+    }
+#endif
+
+    QString caption = QString("<h3>%1</h3><p>%2</p>")
+        .arg(tr("About Chewing Editor"))
+        .arg(tr("Version %1 (%2).").arg(PROJECT_VERSION).arg(PROJECT_GIT_VERSION));
+    QString text = QString("<p>%1</p><p>%2</p><p>%3</p><p>%4</p><p>%5</p><p>%6</p>")
```

Comment:
I see the struggle here... here's a example how I did for another program: https://github.com/qbittorrent/qBittorrent/blob/master/src/gui/about_imp.h#L53

`<p style=\"white-space: pre-wrap;\">` is the key here, then you can start using `\n`